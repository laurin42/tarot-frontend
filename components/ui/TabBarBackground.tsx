// Überprüfe diese Datei und stelle sicher, dass sie keinen Balken erzeugt
// Beispiel für einen korrekten TabBarBackground:
import React from "react";
import { View } from "react-native";

export default function TabBarBackground() {
  return (
    <View
      style={{
        flex: 1,
        backgroundColor: "#111827", // Konsistent mit der App
      }}
    />
  );
}

export function useBottomTabOverflow() {
  return 0;
}
