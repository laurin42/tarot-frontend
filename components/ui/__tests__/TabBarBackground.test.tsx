import React from "react";
import { render } from "@testing-library/react-native";
import TabBarBackground from "@/components/ui/TabBarBackground";

describe("TabBarBackground Component", () => {
  it("renders correctly and matches snapshot", () => {
    const tree = render(<TabBarBackground />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
