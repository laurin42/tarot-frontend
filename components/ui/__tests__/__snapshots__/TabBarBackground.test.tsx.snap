// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBarBackground Component renders correctly and matches snapshot 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "transparent",
      },
      {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      },
    ]
  }
>
  <ViewManagerAdapter_ExpoBlurView
    blurReductionFactor={4}
    experimentalBlurMethod="none"
    intensity={100}
    style={
      {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
    tint="systemChromeMaterial"
  />
</View>
`;
